src/Plugin/Field/FieldType/AddThisItem.php

```diff
@@ -44,9 +41,8 @@ public static function schema(FieldStorageDefinitionInterface $field_definition)
     return array(
       'columns' => array(
         'value' => array(
-          'type' => $field_definition->getSetting('is_ascii') === TRUE ? 'varchar_ascii' : 'varchar',
-          'length' => (int) $field_definition->getSetting('max_length'),
-          'binary' => $field_definition->getSetting('case_sensitive'),
+          'type' => 'varchar',
+          'length' => 255,
```

Comment:
Why is this defined here? I am curious why this isn't in some config schema or something. Was this intentional?